src/test/compile-fail/const-size_of-cycle.rs

```diff
@@ -13,6 +13,7 @@
 struct Foo {
     bytes: [u8; std::mem::size_of::<Foo>()]
     //~^ ERROR unsupported cyclic reference between types/traits detected
+    //~^^ ERROR unsupported cyclic reference between types/traits detected
```

Comment:
The error message is now thrown twice, once when checking if the field type is WF, and once when checking that it is `DynSized`.

I'm not sure how to prevent this, or whether it even matters.